fa25-team-b/cleaners/race_norm_corrections.py

```
@@ -0,0 +1,133 @@
+    Returns
+    -------
+    Returns original df, with modifed dummy cols and 'is_mixed_race' col
+    '''
+    for race in race_name_map:
+        dummy_col = f'dummy_{race}'
+        if dummy_col in df.columns:
+            df[dummy_col] = (
+                df[col]
+                .astype(str)
+                .str.split('/')
+                .apply(lambda x: int(race in x if isinstance(x, list) else False))
```

Ruff (0.14.8)

Function definition does not bind loop variable `race`

(B023)